src/components/Modals/Nfts/NftModal.tsx

```diff
@@ -0,0 +1,229 @@
+            p={{ base: 8, md: 20 }}
+            justifyContent={{ base: 'flex-start', md: 'center' }}
+            bg='blackAlpha.500'
+            transition='all 1s ease-in-out'
+            backdropFilter='auto'
+            backdropBlur='3xl'
+            transform='translate3d(0, 0, 0)'
+            width='full'
+          >
+            <Image
+              width='full'
+              height='auto'
+              objectFit='contain'
+              src={imageUrl}
+              boxShadow='dark-lg'
+              borderRadius='xl'
+              mb={{ base: -14, md: 0 }}
+              onLoad={() => setImageLoaded(true)}
+            />
+          </Flex>
+        </Flex>
+      </Skeleton>
+    )
+  }, [imageLoaded, imageUrl])
+
+  const nftModalOverview = useMemo(() => {
+    return (
+      <Flex flexDir='column' px={8} pb={6} pt={12} bg={modalHeaderBg} gap={4}>
+        <Flex alignItems='center' justifyContent='space-between'>
+          <Flex flexDir='column'>
+            {collectionLink && (
+              <Button
+                as='a'
+                href={collectionLink}
+                target='_blank'
+                rel='noopener noreferrer'
+                justifyContent='flex-start'
+                variant='link'
+                color='blue.200'
+                fontSize='xs'
+                rightIcon={<ArrowRightUp />}
+              >
+                {collectionName}
+              </Button>
+            )}
+            <RawText fontSize='lg' fontWeight='bold'>
+              {name}
+            </RawText>
+          </Flex>
+        </Flex>
+        <StatGroup>
+          {floorPriceEth && (
+            <Stat>
+              <StatLabel>{translate('nft.floorPrice')}</StatLabel>
+              <StatNumber>{floorPriceEth} ETH</StatNumber>
+            </Stat>
+          )}
+          {lastSaleEth && (
+            <Stat>
+              <StatLabel>{translate('nft.lastPrice')}</StatLabel>
+              <StatNumber>{lastSaleEth} ETH</StatNumber>
+            </Stat>
+          )}
+          {rarityDisplay && (
+            <Stat>
+              <StatLabel>{translate('nft.rarity')}</StatLabel>
+              <StatNumber>
+                <Tag colorScheme='purple' variant='solid'>
+                  <TagLeftIcon as={DiamondIcon} />
+                  {rarityDisplay}
+                </Tag>
+              </StatNumber>
+            </Stat>
+          )}
+        </StatGroup>
+      </Flex>
+    )
+  }, [
+    modalHeaderBg,
+    collectionLink,
+    collectionName,
+    name,
+    floorPriceEth,
+    translate,
+    lastSaleEth,
+    rarityDisplay,
+  ])
+
+  const nftModalDetails = useMemo(() => {
+    return (
+      <Tabs display='flex' flexDir='column' position='relative' variant='unstyled' flex={1}>
+        <Box position='relative'>
+          <TabList gap={4} px={8} bg={modalHeaderBg}>
+            <NftTab>{translate('nft.overview')}</NftTab>
+            {/* <NftTab>{translate('nft.properties')}</NftTab> */}
```

Comment:
i'd normally not leave commented code but i think we can get some richer data from a slightly different endpoint in a follow up PR so i'd like to leave this in (as it won't be in git history)